bundles/org.openhab.core/src/main/java/org/openhab/core/library/types/DecimalType.java

```diff
@@ -53,7 +53,7 @@ public DecimalType() {
     public DecimalType(Number value) {
         if (value instanceof QuantityType type) {
             this.value = type.toBigDecimal();
-        } else if (value instanceof HSBType type) {
+        } else if (value instanceof DecimalType type) {
```

Comment:
now uses internal representation, should be faster... As ```DecimalType``` is base class of ```HSBType``` and ```PercentType```, this will apply to more types.

Reply:
Yes, this is only to hit more types